VENDORS/Milesight/AM308/ChirpStack/uplink/converter.json

```diff
@@ -1,39 +1,39 @@
-        "encoder": null,
-        "tbelEncoder": null,
-        "updateOnlyKeys": [
-            "tenantId",
-            "tenantName",
-            "applicationId",
-            "applicationName",
-            "deviceProfileId",
-            "deviceProfileName",
-            "devEui",
-            "devAddr",
-            "fPort",
-            "frequency",
-            "bandwidth",
-            "spreadingFactor",
-            "codeRate",
-            "battery",
-            "pir",
-            "daylight",
-            "confirmed",
-            "gatewayId",
-            "channel",
-            "rfChain",
-            "crcStatus"
-        ]
-    },
-    "additionalInfo": {
-        "description": ""
-    },
-    "edgeTemplate": false
-}
+  "name": "ChirpStack uplink converter for Milesight AM308",
```

Comment:
Please fix formatting to be the same as for other converters - 4-spaces tab size